### ISSUE TITLE
CIFAR-10: Training set and test set for Pretrained checkpoints.

### ISSUE THREAD

**Issue by adderbyte** [POSTECH-CVLab/PyTorch-StudioGAN#195]
Hi,
Thank you for this great job!
Just a quick question, as per the documentation for [CIFAR-10 data ](https://www.cs.toronto.edu/~kriz/cifar.html), there are 50000 train images and 10000 test images. 
For the [checkpoints](https://huggingface.co/Mingguksky/PyTorch-StudioGAN/tree/main/studiogan_official_ckpt/CIFAR10_tailored), did you train on all data (50000+10000) or you reserved the test images? And did you involve the test images at any point in the training pipeline?

Thank you!

**Comment by alex4727** [comment]
Following common practice, we use only training data for all checkpoints and benchmark tables. 
So we do not involve any test images in training pipeline.
